src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -182,5 +212,16 @@ public void writeThrough(Conversation conversation) {
     conversationEntity.setProperty("creation_time", conversation.getCreationTime().toString());
     datastore.put(conversationEntity);
   }
+  
+  
+  public void writeThrough(Profile profile) {
+	Entity profileEntity = new Entity("chat-profile", profile.getId().toString());
+	profileEntity.setProperty("uuid", profile.getId().toString());
+	profileEntity.setProperty("profile_uuid", profile.getProfile().toString());
+	profileEntity.setProperty("about", profile.getAbout());
+	profileEntity.setProperty("creation_time", profile.getCreation().toString());
+	datastore.put(profileEntity);
+}
```

Comment:
Missing two spaces before this curly brace

Reply:
Done.